ganga/GangaCore/GPIDev/Adapters/IVirtualization.py

```diff
@@ -0,0 +1,34 @@
+##########################################################################
+# Ganga Project. http://cern.ch/ganga
+#
+##########################################################################
+from GangaCore.GPIDev.Schema import Schema, Version, SimpleItem
+from GangaCore.GPIDev.Base import GangaObject
+
+class IVirtualization(GangaObject):
+
+    """
+    Contains the interface for all virtualization classes, all virtualization classes should inherit from this object.
+    """
+
+    def __init__(self, imageUrl):
+        super(IVirtualization, self).__init__()
+        self.imageUrl = imageUrl
+
+    _schema = Schema(Version(1, 0), {
+        'imageUrl': SimpleItem(defvalue="", doc='Link to the container image')
```

Comment:
See general comment. Maybe we should at least just change name to "image", rather than imageUrl. Then it will be easier later to extend the list of accepted types without having to migrate anything for users.